.eslintrc

```diff
@@ -11,7 +11,8 @@
     "prettier/prettier": ["error"],
     "import/prefer-default-export": "off",
     "no-plusplus": "off",
-    "class-methods-use-this": "off"
+    "class-methods-use-this": "off",
+    "no-restricted-exports": "off"
```

Comment:
This triggered the modules we exported with default as
export { default } from 'abc/xyz.js'

We were using this pattern at multiple times, so escaped this for now. 
Let me know if I am understanding it wrong, here's the linting rule details: https://eslint.org/docs/latest/rules/no-restricted-exports